src/vue/modules/pre-issuance-form/mixins/manage-pre-issuance.mixin.spec.js

```diff
@@ -0,0 +1,154 @@
+import ManagePreIssuanceMixin from './manage-pre-issuance.mixin'
+
+import { Wallet, base } from '@tokend/js-sdk'
+
+import { mount, createLocalVue } from '@vue/test-utils'
+
+import { FileUtil } from '@/js/utils/file.util'
+
+import * as Api from '../_api'
+import { AssetNotOwnedError, FileCorruptedError } from '../_errors'
+import { Issuance } from '../wrappers/issuance'
+
+const localVue = createLocalVue()
+
+const Component = {
+  template: `<div></div>`,
+  props: ['ownedAssets'],
+}
+
+describe('Manage pre-issuance mixin', () => {
+  let sandbox
+  let wrapper
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox()
+
+    wrapper = mount(Component, {
+      mixins: [ManagePreIssuanceMixin],
+      localVue,
+    })
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+  })
+
+  describe('method', () => {
+    describe('createPreIssuanceRequest', () => {
+      beforeEach(() => {
+        const wallet = new Wallet(
+          'test@mail.com',
+          'SCPIPHBIMPBMGN65SDGCLMRN6XYGEV7WD44AIDO7HGEYJUNDKNKEGVYE',
+          'GDIU5OQPAFPNBP75FQKMJTWSUKHTQTBTHXZWIZQR4DG4QRVJFPML6TTJ',
+          '4aadcd4eb44bb845d828c45dbd68d5d1196c3a182b08cd22f05c56fcf15b153c'
+        )
+        const config = {
+          horizonURL: 'https://test.api.com',
+        }
+
+        Api.initApi(wallet, config)
+      })
```

Comment:
since this is not the integration test, you do not need to init api with a real wallet. you do not need to `initApi()` at all, you can just stub it to return an `ApiCaller` instance.

we did speak about not-stubbing real data, didn’t we?

Reply:
Sorry, I misunderstood you. I didn't think about stubbing `api()` function directly. Now it's fixed.